coa-helm/templates/cbas-api.yaml

```diff
@@ -1,24 +1,29 @@
+{{- if .Values.cbas.coaEnabled }}
 {{ include "terra-batch-libchart.cbas-api-deploy" (list . "coa-helm.cbas-api-deploy") }}
+{{ end }}
 {{ define "coa-helm.cbas-api-deploy" }}
 # Overrides go here
 spec:
-  template:
-    metadata:
-      labels:
-    {{ include "coa-helm.labels" $ | nindent 8 }}
+template:
```

Comment:
Spacing looks wrong here?

Reply:
Hmmm I think my IDE did something funky haha -- should be fixed!